dashboard/app/models/script.rb

```diff
@@ -1179,6 +1179,7 @@ def update_text(script_params, script_text, metadata_i18n, general_params)
             hidden: general_params[:hidden].nil? ? true : general_params[:hidden], # default true
             login_required: general_params[:login_required].nil? ? false : general_params[:login_required], # default false
             wrapup_video: general_params[:wrapup_video],
+            family_name: general_params[:family_name],
```

Comment:
do we need version_year here too?

Reply:
no, because version_year already existed and is located elsewhere, but the logic is a little obfuscated --

`family_name` is its own column in the scripts table, so it gets updated here in the `update_text` method. however, anything that is in the `properties` column (including `version_year`) of the scripts table is updated in [`build_property_hash`](https://github.com/code-dot-org/code-dot-org/blob/eb88ac28b5a61cf0524ab3abad72bb7747a4d5c9/dashboard/app/models/script.rb#L1460-L1482), which is called by `update_text` just below this line (on [line 1182](https://github.com/code-dot-org/code-dot-org/blob/eb88ac28b5a61cf0524ab3abad72bb7747a4d5c9/dashboard/app/models/script.rb#L1182))